scripts/addLiquidity.ts

```diff
@@ -1,7 +1,9 @@
-const { ethers } = require("hardhat");
```

Comment:
We need to update the script to support other network tokens too. As of now, tokens are of mainnet only

Reply:
Should we do it in this PR or another?